Tests/IntegrationTests/App.config

```diff
@@ -5,10 +5,13 @@
         <add key="StsEndpointAddress" value="https://SecureTokenService.test-nemlog-in.dk/SecurityTokenService.svc" />
         <add key="StsCertificatePath" value="Resources\\IntTest_STS.cer" />
         <add key="TokenLifeTimeInMinutes" value="5" />
-        <add key="DotNetWspEntityID" value="https://wsp.oioidws-net.dk" />
-        <add key="DotNetWspHostname" value="https://Digst.OioIdws.Wsp:9090/HelloWorld" />
-        <add key="DotNetWspCertificatePath" value="Resources\\DotNet_WSP.p12" />
         <add key="LocalTokenServiceSigningCertificatePath" value="Resources\\sts.oioidws-net.dk.p12" />
         <add key="LocalTokenServiceEntityId" value="https://sts.oioidws-net.dk" />
+        <add key="AccessTokenIssuerEndpoint" value="https://digst.oioidws.rest.wsp:10002/accesstoken/issue" />
```

Comment:
Det bør nok være `key="DotNetRestAccessTokenIssuerEndpoint"` da der kommer et tilsvarende for for Java